core/tests/bank_forks.rs

```diff
@@ -270,7 +312,7 @@ mod tests {
             if slot == saved_slot as u64 {
                 let options = CopyOptions::new();
                 fs_extra::dir::copy(accounts_dir, &saved_accounts_dir, &options).unwrap();
-                let snapshot_paths: Vec<_> = fs::read_dir(&snapshot_config.snapshot_path)
+                let snapshot_paths = fs::read_dir(snapshot_path)
```

Comment:
nits: (I merged this because `nits` doesn't worth to block this useful pr. Feel free to create mini follow-up pr if this bothers you enough.) this started to be called `.last()` before assignment, so rename `snapshot_paths` => `snapshot_path`

Reply:
Fixed in #10721